src/datatypes/character/Character.hpp

```diff
@@ -107,8 +115,9 @@ namespace spy::character {
             unsigned int intelligencePoints = DEFAULT_INTELLIGENCE_POINTS;
             unsigned int chips = DEFAULT_CHIPS;
             std::set<spy::character::PropertyEnum> properties;
-            std::vector<spy::gadget::Gadget> gadgets;
+            std::vector<std::shared_ptr<spy::gadget::Gadget>> gadgets;
             character::FactionEnum faction = FactionEnum::INVALID;
+            std::optional<util::UUID> wiredBy;
```

Comment:
was hat das mit dem gadget->shared_ptr zu tun?

Reply:
Nichts, wird aber für wiretap with earplugs benötigt und in dem PR direkt mit rein genommen um später unnötiges mergen zu vermeiden.

Reply:
dafür bitte in zukunft eigenen PR aufmachen